src/CSnakes.Tests/PythonStaticGeneratorTests/FormatClassFromMethods.test_optional.approved.txt

```diff
@@ -133,6 +138,19 @@ public static class TestClassExtensions
                 return __return;
             }
         }
+
+        public (long? , string) TestOptionalTuple((long? , string) a)
+        {
+            using (GIL.Acquire())
+            {
+                this.logger?.LogDebug("Invoking Python function: {FunctionName}", "test_optional_tuple");
+                PyObject __underlyingPythonFunc = this.__func_test_optional_tuple;
+                using PyObject a_pyObject = PyObject.From(a)!;
+                using PyObject __result_pyObject = __underlyingPythonFunc.Call(a_pyObject);
+                var __return = __result_pyObject.BareImportAs<(long? , string), global::CSnakes.Runtime.Python.PyObjectImporters.Tuple<long?, string, global::CSnakes.Runtime.Python.PyObjectImporters.Runtime<long?>, global::CSnakes.Runtime.Python.PyObjectImporters.String>>();
```

Comment:
Unfortunately, `global::CSnakes.Runtime.Python.PyObjectImporters.Runtime<long?>` here means that `long?` is imported via run-time reflection whereas normally its importer would be resolved at build-time. This introduces subtle differences in when Native AOT would be supported.